src/App.tsx

```diff
@@ -1361,116 +1347,7 @@ function App(props: RouteComponentProps) {
                     className={showAbout ? 'about-modal-container' : 'about-modal-container-hidden'}
                     onClick={() => setShowAbout(false)}
                 />
-                {!isMinimalMode && (
-                    <div className={showAbout ? 'about-modal' : 'about-modal-hidden'}>
-                        <button className="about-modal-close-button" onClick={() => setShowAbout(false)}>
-                            <svg
-                                xmlns="http://www.w3.org/2000/svg"
-                                width="30"
-                                height="30"
-                                viewBox="0 0 16 16"
-                                strokeWidth="2"
-                                stroke="none"
-                                fill="currentColor"
-                                strokeLinecap="round"
```

Comment:
Thank you for moving this to a separate file! The App.tsx file was becoming too long, so this is great.